_source/_includes/log-shipping/multiline-logs-filebeat.md

```diff
@@ -0,0 +1,81 @@
+## Multiline logs
+
+
+If your original logs span multiple lines, you may find that they arrive in your Logz.io account split into several partial logs. You have the option to use either an explicit configuration or an annotation to configure Filebeat to concatenate multiline logs and avoid having them separated into multiple log documents.
+
+You will need to provide a regex expression that can identify the first log in a multiline log. Filebeat will concatenate the log lines that follow until it identifies the next log that matches the regex expression. In other words, there is no explicit regex expression to demarcate the _end_ of a multiline log.
+
+This behavior is managed differently depending on your deployment method:
+
+* If you are using autodiscover hints & annotations, add an annotation to your deployment.
+* If you are not using autodiscover, use an explicit configuration.
+
+
+
+### Using an explicit configuration to concatenate multiline logs
+
```

Comment:
Please add somewhere in this section a reference to Filebeat's page that has further and needed explanations on the options those configs have
https://www.elastic.co/guide/en/beats/filebeat/7.8/multiline-examples.html